### PR TITLE
Feature/consolidate naming

### PR DESCRIPTION
**Pull request by jsauter**
Renaming of the alb and asg to be more concise.

### CODE REVIEW

File: cumulus/steps/ec2/dns.py [hunk 1]
```diff
@@ -1,47 +1,43 @@
...
         self.dns_name = dns_name
 
     def handle(self, chain_context):
         template = chain_context.template
 
-        name = 'AlbAlias%s' % chain_context.instance_name
-
         template.add_resource(route53.RecordSetGroup(
-            "Route53Records",
+            "Route53Record",
```
Comment (brettswift) [inline cumulus/steps/ec2/dns.py]: This name seems interesting..  we may just be adding a single recordset but the resource is a group that can take multiple.

File: cumulus/steps/ec2/ingress_rule.py [hunk 2]
```diff
@@ -9,19 +9,17 @@ class IngressRule(step.Step):
...
 
         self.port_to_open = port_to_open
         self.cidr = cidr
-        self.name = name
 
     def handle(self, chain_context):
         template = chain_context.template
         template.add_resource(ec2.SecurityGroupIngress(
-            self.name,
+            self.name % self.port_to_open,
```
Comment (brettswift) [inline cumulus/steps/ec2/ingress_rule.py]: why is self.name being set here and not in the constructor?   flake8 should be catching this.. doesn't it?

Comment (brettswift) [reply cumulus/steps/ec2/ingress_rule.py]: right self.name comes in from the base class.   It's kind of nasty to have some names go in with a replacement token that the class has to handle later on.   If I want to just use self.name later on I have to think about replacing it.

__init__()
```
name = 'Port%sIngressRule'  % self.port_to_open
 step.Step.__init__(self, name=name)

```

File: cumulus/chain/step.py [hunk 3]
```diff
@@ -6,8 +6,12 @@ class Step:
     Define an interface for handling requests.
     """
 
-    def __init__(self):
-        pass
+    def __init__(self, name='UnNamed'):
```
[the remark this answers is not in the document]
Comment (brettswift) [reply cumulus/chain/step.py]: I mean don't set a default.

**Comment by jsauter** [comment]
Closign this pull request because something went wonky.

**Pull request closed without merging**